controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaDataTests.java

```diff
@@ -196,7 +196,7 @@ public void setup() throws Exception {
                                                                         .controllerURI(URI.create("tcp://localhost"))
                                                                         .build());
         restServer = new RESTServer(controller, mockControllerService, authManager, serverConfig,
-                connectionFactory);
+                connectionFactory, ClientConfig.builder().controllerURI(URI.create("tcp://localhost")).build());
```

Comment:
nit: you dont need to give controller uri here for this test..

Reply:
removed it